.github/workflows/ci-workflow.yml

```diff
@@ -1,6 +1,6 @@
 name: Apollo Server Plugin CI
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Would you be willing to also add scheduling here? Because it is a plugin, we don't have anything that would force the agent to run these and if we got a large gap of time we can have random breakages like we just stumbled into.

Would be cool to run once a week or once every-other week.

Going to approve this in case you don't want to muddle this PR.